features/kafka/producer/src/main/java/org/opennms/features/kafka/producer/model/CollectionSetProtos.java

```diff
@@ -1,8 +1,8 @@
 /*******************************************************************************
  * This file is part of OpenNMS(R).
  *
- * Copyright (C) 2018 The OpenNMS Group, Inc.
- * OpenNMS(R) is Copyright (C) 1999-2018 The OpenNMS Group, Inc.
+ * Copyright (C) 2021 The OpenNMS Group, Inc.
```

Comment:
not to nitpick but this should be 2018-2021 then  :D

Reply:
hmm, I didn't know that. I'm always using 1999-current year for all files.

Reply:
Well, it generally should be the first time the file was made through the last time it was edited.  So in _this_ case it started at 2018 (presumably)

Reply:
Okay.